pylmod/gradebook.py

```diff
@@ -0,0 +1,341 @@
+"""
+
+"""
+import csv
+import json
+import logging
+import time
+
+from base import Base
+
+
+log = logging.getLogger(__name__)
+
+
+class GradeBook(Base):
+    """
+
```

Comment:
Please add an actual docstring.  Preferably the google style class doc string, but a simple description is fine too